.gitignore

```diff
@@ -12,8 +12,36 @@ src/dpsvalidationservice/dpsvalidationservice.iml
 
 src/dpsvalidationservice/target/
```

Comment:
Check if this is required. The other projects don;t use it. They just have "target/"

Reply:
nope intellij

Reply:
OK. Thanks